Core/SiteRating.swift

```diff
@@ -55,13 +55,23 @@ public class SiteRating {
             
         return nil
     }
-    
-    public var containsMajorTracker: Bool {
-        return trackersDetected.contains(where: { $0.key.fromMajorNetwork } )
+
+    public var uniqueMajorTrackersDetected: Int {
```

Comment:
I still think the label for these is misleading. It's the number of unique major tracker networks detected not number of trackers. Sorry! If `uniqueMajorTrackerNetworksDetected` is too much how about `uniqueMajorNetworksDetected`? Happy with any label that captures the meaning!